site/guide/working-with-model-inventory.qmd

```diff
@@ -0,0 +1,51 @@
+---
+title: "Working with the model inventory"
+date: last-modified
+listing:
+  - id: model-inventory-listing
+    type: grid
+    max-description-length: 250
+    sort: false
+    fields: [title, description]
+    contents:
+      - register-models-in-model-inventory.qmd
+      - edit-model-inventory-fields.qmd
+      
+---
+The model inventory holds all models. Within the model inventory, register new models and edit fields. Organize the inventory by searching, filtering and sorting models. Configure the inventory by managing column layout and choosing card or table view.
+
+## Register models and edit fields
+See these guides on how to add new models and edit their fields.
+
+:::{#model-inventory-listing}
+:::
+
+## Search, filter, and sort models
+Use the provided tools to navigate through the model inventory.
+
+### Search models
+Search for models using **{{< fa magnifying-glass >}} Search**. 
+![](search-demo.gif){width=80% fig-alt="An animation that shows how to search in the model inventory."}
+
+
+### Filter models
+Use **{{< fa filter >}} Filter** to filter models by business units, developers, model tiering, model status, owners, use, validators, or vendor model.
+![](filter-demo.gif){width=80% fig-alt="An animation that shows how to use the filter in the model inventory."}
+
+
+### Sort models
+Use **{{< fa arrow-up-wide-short >}} Sort** to sort results by certain criteria.
+![](sort-demo.gif){width=80% fig-alt="An animation that shows how to use the sort feature in the model inventory."}
+
+
+## Manage inventory layout
```

Comment:
I would rework this section as well, maybe break it down into (h4 header): 

- Since table view is the default view, I would just remove the filler text about switching and talk about how you can customize the columns under this main section header
- Also, how do I GET to the inventory to customize? You forgot the step-by-step! 

#### Display cards view
- Instructions on how to toggle display card view
- Make sure they're step-by-step ;)